org.torproject.torbrowser.yml

```diff
@@ -0,0 +1,50 @@
+app-id: org.torproject.torbrowser
```

Comment:
Since this is not a torbrowser, but a 3rd-party wrapper, not developed by torproject, consider giving it appropriate ID.